app/src/main/AndroidManifest.xml

```diff
@@ -4,9 +4,9 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.goryachok.forecastapp">
 
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Please revert that. Export and use only 1 style settings file.

Reply:
Ok